index.html

```diff
@@ -6,10 +6,73 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
 </head>
 
 <body>
+    <div class="container">
+        <section class="portada">
+            <img class="portada-img" src="ap.png" alt="ap.png">
+            <div class="flex-box">
+                <div class="box-header">
+                    <h1 class="header">GOT MARKETING? ADVANCE YOUR BUSINESS INSIGHT</h1>
+                </div>
+                <div class="box-parrafo">
+                    <P class="parrafo">Fill out the form and receive our award winning newsletter.</P>
+                </div>
+            </div>
+        </section>
+        <section class="datos">
+            <p class="entrada">Name</p>
+            <input type="name" id="nombre">
+            <p class="correo">Email</p>
+            <input type="email" id="mail">
+            <button id="sing" type="submit">Sing me up</button>
+        </section>
+    </div>
+    <div class="responsive__container">
+        <section class="responsive__portada">
+            <div class="responsive__flex-box">
+                <div class="responsive__box-header">
+                    <h1 class="responsive__header">GOT MARKETING? ADVANCE YOUR BUSINESS INSIGHT</h1>
+                </div>
+                <div class="responsive__box-parrafo">
+                    <P class="responsive__parrafo">Fill out the form and receive our award winning newsletter.</P>
+                </div>
+            </div>
+        </section>
+        <section class="responsive__datos">
```

Comment:
Great Albin, I can suggest check out other way to manage the `responsive design`, this type of code if for use the same structure of `html` for both design (maybe more, mobile, desktop, tv, tablet..) you can use the CSS classes for this :smiley: